assets/javascripts/discourse/widgets/qa-post.js.es6

```diff
@@ -23,7 +23,7 @@ export default createWidget('qa-post', {
     const user = this.currentUser;
 
     if (post.get('topic.voted')) {
-      return bootbox.alert(I18n.t('vote.already_voted'));
+      return bootbox.alert(I18n.t('vote.user_over_limit'));
```

Comment:
We'll need to add a seperate ``user_over_limit`` string in the ``client.en.yml`` file in ``config/locales``. There are seperate locale files for the server and client.

Reply:
Added.